README.md

```diff
@@ -80,18 +80,13 @@ In the following table you can find what are supported Kubernetes versions for
 each KubeOne version. KubeOne versions that are crossed out are not supported.
 It's highly recommended to use the latest version whenever possible.
 
-| KubeOne version | 1.14 | 1.13 | 1.12 | Supported providers                                |
```

Comment:
This table, do we really want to add every release, even nothing changed besides version number? I propose to squeeze it to the format when combination of `1.14 | 1.13 | Supported providers` will be uniq. Basically `v0.6.0` will turn into `v0.6.0+` and everything about it should be dropped.

Reply:
I'd probably drop the patch release, so we have just `v0.5`, `v0.6`, `v0.7`, etc. WDYT?

Reply:
I see it as having

```md
| v0.6.0+         | +    | +    | AWS, DigitalOcean, GCE, Hetzner, Packet, OpenStack |
| v0.5.0          | +    | +    | AWS, DigitalOcean, GCE, Hetzner, OpenStack         |
```
is enough, until get new providers. Because repeating same lines again and again looks like visual noise.

IMHO this table is not a "releases list". It's here to provide info about main features support.

Reply:
I don't have strong opinions on this, so let's just proceed as you proposed.